kotlin-analysis-plugin/README.md

```diff
@@ -1,3 +1,11 @@
 # Kotlin-Analysis plugin
 
-The plugin to run the required functionality from the CLI
\ No newline at end of file
+The plugin to run the required functionality from the CLI
+
+### Usage
+
+To run analysis execute task 'cli' with analysis runner name and args.
+
+``` 
+gradle :kotlin-analysis-plugin:cli -Prunner="kotlin-clones-analysis" -Pinput="path/to/dir/with/projects" -Poutput="path/to/dir/with/results"
```

Comment:
The output folder will be created if it does not exist? If not, I think we should correct it

Reply:
It is created.